ra/ra.go

```diff
@@ -358,6 +360,34 @@ func (ra *RegistrationAuthorityImpl) checkPendingAuthorizationLimit(ctx context.
 	return nil
 }
 
+func (ra *RegistrationAuthorityImpl) checkInvalidAuthorizationLimit(ctx context.Context, regID int64) error {
+	limit := ra.rlPolicies.InvalidAuthorizationsPerAccount()
+	saGRPC, ok := ra.SA.(bgrpc.StorageAuthorityClientWrapper)
```

Comment:
Could you add a comment about this cast, or maybe change the name of `ok` to make it clearer that you're doing this cast as a means of checking if the RPC was over gRPC?